src/types/Graph.ts

```diff
@@ -1,8 +1,13 @@
 import { Duration, Layout, BadgeStatus } from './GraphFilter';
 import Namespace from './Namespace';
 
```

Comment:
You can remove the comment at ServiceGraphPage:15 and move it here.

Reply:
Done!